README.md

```diff
@@ -9,7 +9,22 @@ to connect to multiple different account sources.
 More information about SSSD can be found on its project page -
 https://pagure.io/SSSD/sssd/
 
-## Building and installation
+## Downloading SSSD
+SSSD is shipped as a binary package by most Linux distributions. If you
+want to obtain the latest source files, please navigate to the
+[Releases folder on pagure](https://releases.pagure.org/SSSD/sssd/)
+
+## Releases
+SSSD maintains two release streams - stable and LTM. Releases designated as
+LTM are long-term maintenance releases and will see bugfixes and security
+patches for a longer time than other releases.
+
+The list of all releases is maintained together with [SSSD documentation](https://docs.pagure.org/SSSD.sssd/users/releases.html)
+
+The latest stable release is [1.15.1](https://docs.pagure.org/SSSD.sssd/users/relnotes/notes_1_15_1.html)
+The latest LTM release is 1.13.4.
```

Comment:
Is it a problem to change README? This would be documented in the release process.